app/javascript/packs/application.js

```diff
@@ -7,7 +7,10 @@ import Rails from "@rails/ujs"
 import Turbolinks from "turbolinks"
 import * as ActiveStorage from "@rails/activestorage"
 import "channels"
+import "bootstrap"
```

Comment:
Use single quote here for string as there is no special characters and string interpolation in it